tests/unit/client/client.test.ts

```diff
@@ -30,4 +52,70 @@ describe('HttpClient', () => {
 
     expect(test.data).toBe('');
   });
+
+  it('Should transform dates in the response when authenticating a user', async () => {
+    nock(`${host}${AUTH_BASE}`)
+      .post('/oauth1/tokens')
+      .reply(200, authenticationResponse);
+
+    const response = await exh.auth.authenticate({
+      email: 'string',
+      password: 'string',
+    });
+
+    expect(response).toStrictEqual({
+      ...authenticationResponse,
+      creationTimestamp: new Date(authenticationResponse.creationTimestamp),
+      updateTimestamp: new Date(authenticationResponse.updateTimestamp),
+    });
+  });
+
+  it('Should transform dates in the response when creating an account', async () => {
```

Comment:
(NO NEED TO CHANGE! 😄) Just to clarify, from my side the intent was to adjust the existing test for the `sdk.users.createAccount` in `tests\unit\users\usersService.test.ts` called `should register a new user`.

I know this is not from the perspective of the changes but rather from the outside. I'm guessing those different views might have caused some confusion. (Same goes for the `authencticate` and `confirmMfa` method tests of the oAuth1 client btw)